### PR TITLE
[ProblemChild] Update files to add Serverless support to the package

### PR DESCRIPTION
**Pull request by sodhikirti07**
## What does this PR do?
This PR adds serverless support to the `Living Off the Land` advanced analytics package.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## How to test this PR locally

- Follow the steps to spin a Serverless project using `elastic-package`
- Make sure the format spec for `manifest.yml` is `3.0.0`
- Test if all the assets i.e. trained model, ingest pipelines, anomaly detection jobs and detection rules are getting installed successfully.

## Screenshots

- Trained model and ingest pipelines:
image

image

- Anomaly detection jobs installed and started:
image

image

- Security rules are enabled:
image

### CODE REVIEW

File: packages/problemchild/manifest.yml [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 description: "ML solution package to detect Living off the Land (LotL) attacks in your environment. Requires a Platinum subscription."
 type: integration
 categories:
   - security
   - advanced_analytics_ueba
-release: ga
 conditions:
-  kibana.version: "^8.0.0"
+  kibana:
+    version: '^8.0.0'
```
Comment (ajosh0504) [inline packages/problemchild/manifest.yml:14]: Is this minimum Stack version compatible with Package Spec v3?

Comment (sodhikirti07) [reply packages/problemchild/manifest.yml:14]: There's no [mention](https://github.com/elastic/package-spec/issues/539) of minimum stack version in the package spec for v3.

**Review by qn895 — APPROVED** [review]

**Comment by qn895** [comment]
License change LGTM 🎉

File: packages/problemchild/manifest.yml (hunk 1 above)
Comment (sodhikirti07) [reply packages/problemchild/manifest.yml:14]: I checked with the serverless team and they confirm that we don't require a minimum Kibana version for serverless packages and they'll be based on spec versions.

**Review by ajosh0504 — APPROVED** [review]
LGTM!

**Pull request merged**